packages/overlay-root/src/active-overlay.ts

```diff
@@ -272,7 +272,8 @@ export class ActiveOverlay extends LitElement {
     };
 
     private onSlotChange(): void {
-        this.updateOverlayPosition();
+        // Edge needs a little time to update the DOM before computing the layout
+        requestAnimationFrame(() => this.updateOverlayPosition());
```

Comment:
I really wish there were a way for us to test that this did regress... Edgium can't come fast enough! De we already have a ticket for getting Browser Stack or Sauce Labs included?